fire.js

```
@@ -1,146 +1,97 @@
+                Buffer2[x][y-1] = (Buffer1[x][y]+ Buffer1[x][y+1])/2 -4;
+            }
+            var val = Buffer2[x][y];
+            var r = val;
+            var g = val * 0.5;
+            var b = 0;
+            setPixel(imageData, x, y, r, g, b, 0xff);
+        }
+    }
+    ctx.putImageData(imageData, 0, 0);
+
+    for (var x = 0; x < width; x++){
```

Biome (1.9.4)

[error] Shouldn't redeclare 'x'. Consider to delete it or rename it.

'x' is defined here:


(lint/suspicious/noRedeclare)